setup.cfg

```diff
@@ -101,7 +101,7 @@ testing =
     pytest-asyncio
     pytest-mock
     pytest-timeout
-    pytest-httpx
+    pytest-httpx==0.30.0
```

Comment:
Can you explain why we're pinning this version, in case we need to check later if an upgrade is possible?